tests/system/test_cluster/test_agent_groups/test_agent_groups_default.py

```diff
@@ -0,0 +1,129 @@
+"""
+copyright: Copyright (C) 2015-2022, Wazuh Inc.
+           Created by Wazuh, Inc. <info@wazuh.com>.
+           This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+type: system
+brief: Verify that the agent connects correctly to the cluster and that when it has no specific
+       configuration, the agent belongs to the default group.
+.
+tier: 0
+modules:
+    - cluster
+components:
+    - manager
+    - agent
+path: /tests/system/test_cluster/test_agent_groups/test_agent_groups_default.py
+daemons:
+    - wazuh-db
+    - wazuh-clusterd
+os_platform:
+    - linux
+os_version:
+    - Arch Linux
+    - Amazon Linux 2
+    - Amazon Linux 1
+    - CentOS 8
+    - CentOS 7
+    - CentOS 6
+    - Ubuntu Focal
+    - Ubuntu Bionic
+    - Ubuntu Xenial
+    - Ubuntu Trusty
+    - Debian Buster
+    - Debian Stretch
+    - Debian Jessie
+    - Debian Wheezy
+    - Red Hat 8
+    - Red Hat 7
+    - Red Hat 6
+references:
+    - https://github.com/wazuh/wazuh-qa/issues/2505
+tags:
+    - cluster
+"""
+import os
+import time
+
+import pytest
+
+from socket import timeout
+from system.test_cluster.test_agent_groups.common import register_agent
+from wazuh_testing.tools import WAZUH_PATH
+from wazuh_testing.tools.system import HostManager
+from system import (AGENT_NO_GROUPS, AGENT_STATUS_ACTIVE, AGENT_STATUS_NEVER_CONNECTED, AGENT_GROUPS_DEFAULT, 
+                    check_agent_groups, check_agent_status, restart_cluster, check_keys_file)
+
+
+# Hosts
+test_infra_managers = ["wazuh-master", "wazuh-worker1", "wazuh-worker2"]
+test_infra_agents = ["wazuh-agent1"]
+
+inventory_path = os.path.join(os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__)))),
+                              'provisioning', 'enrollment_cluster', 'inventory.yml')
+host_manager = HostManager(inventory_path)
+local_path = os.path.dirname(os.path.abspath(__file__))
+tmp_path = os.path.join(local_path, 'tmp')
+timeout = 5
+
+# Tests
+@pytest.mark.parametrize("test_infra_managers",[test_infra_managers])
+@pytest.mark.parametrize("test_infra_agents",[test_infra_agents])
+@pytest.mark.parametrize("host_manager",[host_manager])
+@pytest.mark.parametrize("agent_target", test_infra_managers)
+def test_agent_default_group(agent_target, clean_environment, test_infra_managers, test_infra_agents, host_manager):
```

Comment:
Change the module name and the path with the new module name. Ex: test_agent_added_the_default_group. 

Also, when you change it, you should change the path in the documentation.

Reply:
Changed name to test_agent_default_group_added.
Did not add to path in documentation since it was removed as requested